universitas-negeri-semarang-fakultas-matematika-dan-ilmu-pengetahuan-alam.csl

```diff
@@ -149,8 +149,8 @@
               </else>
             </choose>
           </if>
-          <else-if type="thesis">
-            <text variable="genre" suffix="."/>
+          <else-if type="thesis"><!--Note digunakan untuk mensitasi skripsi, thesis dan disertasi-->
+            <text variable="note" suffix="."/>
```

Comment:
I am not able to test right now, but according to Mendeley, since versrion
1.16.2, "Type" should map to "Genre" for theses. Is that not the case?
In that case, you can add fields to item types under options.

On Mon, Sep 12, 2016 at 12:43 AM, Alberto Krishna Winning Kusuma <
notifications@github.com> wrote:

> In universitas-negeri-semarang-fakultas-matematika-dan-ilmu-
> pengetahuan-alam.csl
> <https://github.com/citation-style-language/styles/pull/2202#discussion_r78314964>
> :
>
> > @@ -149,8 +149,8 @@
> >                </else>
> >              </choose>
> >            </if>
> > -          <else-if type="thesis">
> > -            <text variable="genre" suffix="."/>
> > +          <else-if type="thesis"><!--Note digunakan untuk mensitasi skripsi, thesis dan disertasi-->
> > +            <text variable="note" suffix="."/>
>
> @rmzelle <https://github.com/rmzelle>, Right, Thesis should have a
> "Genre" field. But, how to fill Genre variable in Mendeley Desktop? Because
> I just found type, I did not find genre in Mendeley Desktop.
> [image: cuplikan layar dari 2016-09-12 11-32-57]
> <https://cloud.githubusercontent.com/assets/11894323/18424876/0e29c276-78de-11e6-80c5-8c5d4ede2d33.png>
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/citation-style-language/styles/pull/2202/files/154aa796c25ec74969714f6b2b7a01d790821916#r78314964>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAH32mih2qXv1S0qxYT2myM3dLd8DyAmks5qpNhrgaJpZM4J0v3P>
> .
>



-- 
Sebastian Karcher, PhD
www.sebastiankarcher.com